recipe/recipe.yaml

```
@@ -0,0 +1,70 @@
+context:
+  name: libmemcached
+  series: "1.0"
+  version: "1.0.18"
+
+package:
+  name: ${{ name }}
+  version: ${{ version }}
+
+source:
+  url: https://launchpad.net/${{ name }}/${{ series }}/${{ version }}/+download/${{ name }}-${{ version }}.tar.gz
+  sha256: e22c0bb032fde08f53de9ffbc5a128233041d9f33b5de022c0978a2149885f82
+  patches:
+    # thanks to http://stackoverflow.com/questions/27004144/how-can-i-install-libmemcached-for-mac-os-x-yosemite-10-10-in-order-to-install-t
+    - if: osx
+      then:
+        - osx.configure.patch
+        - osx.memflush.patch
+    # Taken from this bug report
+    # ref: https://bugs.launchpad.net/libmemcached/+bug/1663985
+    - if: linux
+      then:
+        - libmemcached-build.patch
+
+build:
+  number: 1004
+  skip:
+    - win
+
+requirements:
+  build:
+    - ${{ compiler('c') }}
+    - ${{ compiler('cxx') }}
+    - ${{ stdlib('c') }}
+    - make
+    - if: unix
+      then:
+        - pkg-config
+        - libtool
+        - gnuconfig
+  host:
+    - glib
+    - if: linux and (build_platform != target_platform)
+      then: ${{ cdt("libuuid") }}
```

```suggestion
    - if: linux
      then: libuuid
```
Does this not work?  `libuuid` is in conda-forge.

---

Thank you for your comment! @danielnachun 
No, it had no effect at all, unfortunately. 😢